config/services.yaml

```diff
@@ -4,6 +4,7 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices.html#use-parameters-for-application-configuration
 parameters:
+    photos_directory: '%kernel.project_dir%/public/build/images'
```

Comment:
No you shouldn't put your images in the build directory, because yarn could delete everything in it.
Put them in directory "public/uploads/..." and don't call the parameter photos_directory => they're not photos